README.md

```diff
@@ -132,7 +151,7 @@ if err == nil {
 }
 ```
 
-Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a userID.
+Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a RememberMeID.
```

Comment:
_rememberMeId_, or _userId_?
Looks like we've referred to this as _userId_ elsewhere.  I don't why there would be a difference, or why we can't just stick to one name.

Can anyone shed any light?

If it was _userId_ before, maybe we should just leave it.

Reply:
As discussed, this will be RememberMeID everywhere now - [SDK-572](https://lampkicking.atlassian.net/browse/SDK-572)